source/eitri/requirements.txt

```diff
@@ -2,3 +2,5 @@ docker-py==0.4.0
 retrying==1.2.3
 psycopg2==2.4.5
 clingon==0.1.4
+GeoAlchemy2==0.2.4
+SQLAlchemy==1.0.4
```

Comment:
I think SqlAlchemy is also needed (even if the dependency might be carried by GeoAlchemy).

You're right for alembic, it has been added